<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <meta name="author" content="Andreas Mock" />
  <title>Einführung in R Programming</title>
  <style type="text/css">code{white-space: pre;}</style>
  <style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
  </style>
  <link href="site_libs/slidy-2/styles/slidy.css" rel="stylesheet" />
  <script src="site_libs/slidy-2/scripts/slidy.js"></script>
  <script src="site_libs/slidy_shiny-1/slidy_shiny.js"></script>
</head>
<body>
<div class="slide titlepage">
  <h1 class="title">Einführung in R Programming</h1>
  <h1 class="subtitle">Tag 5 - RNA-seq Analyse</h1>
  <p class="author">
Andreas Mock
  </p>
  <p class="date">16.06.2020</p>
</div>
<div id="kursmaterial-zur-rna-seq-analyse" class="slide section level2">
<h1>Kursmaterial zur RNA-seq Analyse</h1>
<p>Das Kursmaterial des heutigen Tages entspricht in großen Teilen Kapitel 8 des hervorragenden Buchs <em>Modern Statistics for Modern Biology</em> von Susan Holmes und Wolfgang Huber.</p>
<p><a href="https://www.huber.embl.de/msmb/Chap-CountData.html">Link zu Kapitel 8</a></p>
</div>
<div id="einführung" class="slide section level2">
<h1>Einführung</h1>
<p>An diesem Kurstag möchten wir uns mit der Auswertung von Daten aus einer RNA Sequenzierung (RNA-seq) beschäftigen.</p>
<p>RNA-seq Daten sind so genannte <code>high-throughput count</code> Daten, in denen wir in vielen parallelen Analysen zählen, wieviele reads pro Gen oder Transkript im Sample detektiert wurden.</p>
<ul>
<li><strong>Sequenzierungsbibliothek</strong>: alle Nukleotide eines Samples, welche den Input für unser Sequenzierungsexperiment darstellen</li>
<li><strong>Fragmente</strong>: Da der aktuelle technologische Goldstandard (Illumina Sequenzierung) nur Nukleotide von wenigen hundert Basenpaaren analysieren kann, müssen diese vor der Analyse fragmentiert werden</li>
<li><strong>Read</strong>: Sequenz welche von einem Fragment erhoben wird. Ca. 150 bp bei Illumina. Bei RNA-seq und doppelsträngiger cDNA bezieht sich dies auf jeweils einen Strang</li>
</ul>
<p>Zeitliche Abfolge: 1. Sequenzierung &gt; 2. Counting</p>
</div>
<div id="benötigte-pakete" class="slide section level2">
<h1>Benötigte Pakete</h1>
<ul>
<li><code>pasilla</code>: Paket, welches die Bespieldaten enthält</li>
<li><code>DESeq2</code>: DAS Paket für die Analyse von RNA-seq Daten</li>
<li><code>pheatmap</code>: Paket zur Erstellung von Heatmaps (sehr gute Alternative <code>ComplexHeatmap</code> Paket)</li>
<li><code>broom</code>: Paket für Ergebnisstabellen</li>
</ul>
</div>
<div id="beispieldatensatz" class="slide section level2">
<h1>Beispieldatensatz</h1>
<p>Unseren Bespieldatensatz beziehen wir aus dem Paket <code>pasilla</code></p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a>fn &lt;-<span class="st"> </span><span class="kw">system.file</span>(<span class="st">&quot;extdata&quot;</span>, <span class="st">&quot;pasilla_gene_counts.tsv&quot;</span>,</span>
<span id="cb1-2"><a href="#cb1-2"></a>                  <span class="dt">package =</span> <span class="st">&quot;pasilla&quot;</span>, <span class="dt">mustWork =</span> <span class="ot">TRUE</span>)</span>
<span id="cb1-3"><a href="#cb1-3"></a></span>
<span id="cb1-4"><a href="#cb1-4"></a>counts &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(<span class="kw">read.csv</span>(fn, <span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>, <span class="dt">row.names =</span> <span class="st">&quot;gene_id&quot;</span>))</span></code></pre></div>
<p>Sequenzierungsdaten sollten als Matrix gespeichert sein und nicht als Dataframe bzw. Tibble!</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1"></a><span class="kw">dim</span>(counts)</span></code></pre></div>
<pre><code>## [1] 14599     7</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a>counts[<span class="dv">1</span><span class="op">:</span><span class="dv">3</span>,]</span></code></pre></div>
<pre><code>##             untreated1 untreated2 untreated3 untreated4 treated1 treated2
## FBgn0000003          0          0          0          0        0        0
## FBgn0000008         92        161         76         70      140       88
## FBgn0000014          5          1          0          0        4        0
##             treated3
## FBgn0000003        1
## FBgn0000008       70
## FBgn0000014        0</code></pre>
<p>Diese Matrix wird auch <code>count table</code> genannt. Wir haben Countdaten zu 14,599 Genen von 7 Samples. Hierbei handelt es sich um die “raw” Counts.</p>
</div>
<div id="herausforderungen-der-datenanalyse" class="slide section level2">
<h1>Herausforderungen der Datenanalyse</h1>
<ul>
<li>großer Wertebereich (0-Millionen)</li>
<li>Die nicht-negativen Count Daten (Integers) folgen keine symmetrischen Verteilung und damit keiner! Normalverteilung</li>
</ul>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1"></a><span class="kw">plot</span>(<span class="kw">density</span>(counts), <span class="dt">xlim=</span><span class="kw">c</span>(<span class="dv">0</span>,<span class="dv">10000</span>), <span class="dt">xlab =</span> <span class="st">&quot;Counts&quot;</span>, <span class="dt">main =</span> <span class="st">&quot;&quot;</span>, <span class="dt">bty=</span><span class="st">&quot;none&quot;</span>)</span></code></pre></div>
<p><img src="pres_day5_files/figure-slidy/unnamed-chunk-3-1.png" width="384" /></p>
<ul>
<li>Im Count Daten zwischen Samples zu vergleichen bedarf es einer Normalisierung. Faktoren sind hier u.a. die Sequenzierungsbibliothek, Sequenzierungstiefe, Genlänge, GC content.</li>
</ul>
</div>
<div id="beispieldatensatz---informationen-zu-samples-und-zum-experiment" class="slide section level2">
<h1>Beispieldatensatz - Informationen zu Samples und zum Experiment</h1>
<p>RNAi Knockdown Experiment des Splicingfaktors <em>pasilla</em> in D. melanogaster. Zwei Konditionen: treated und untreated. Den Annotationfile laden wir mit:</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1"></a>annotationFile &lt;-<span class="st"> </span><span class="kw">system.file</span>(<span class="st">&quot;extdata&quot;</span>,</span>
<span id="cb7-2"><a href="#cb7-2"></a>  <span class="st">&quot;pasilla_sample_annotation.csv&quot;</span>,</span>
<span id="cb7-3"><a href="#cb7-3"></a>  <span class="dt">package =</span> <span class="st">&quot;pasilla&quot;</span>, <span class="dt">mustWork =</span> <span class="ot">TRUE</span>)</span>
<span id="cb7-4"><a href="#cb7-4"></a>pasillaSampleAnno &lt;-<span class="st"> </span>readr<span class="op">::</span><span class="kw">read_csv</span>(annotationFile)</span>
<span id="cb7-5"><a href="#cb7-5"></a></span>
<span id="cb7-6"><a href="#cb7-6"></a>pasillaSampleAnno &lt;-<span class="st"> </span><span class="kw">mutate</span>(pasillaSampleAnno,</span>
<span id="cb7-7"><a href="#cb7-7"></a>condition &lt;-<span class="st"> </span><span class="kw">factor</span>(condition, <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;untreated&quot;</span>, <span class="st">&quot;treated&quot;</span>)),</span>
<span id="cb7-8"><a href="#cb7-8"></a>type &lt;-<span class="st"> </span><span class="kw">factor</span>(<span class="kw">sub</span>(<span class="st">&quot;-.*&quot;</span>, <span class="st">&quot;&quot;</span>, type), <span class="dt">levels =</span> <span class="kw">c</span>(<span class="st">&quot;single&quot;</span>, <span class="st">&quot;paired&quot;</span>)))</span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1"></a>pasillaSampleAnno</span></code></pre></div>
<pre><code>## # A tibble: 7 x 8
##   file  condition type  `number of lane~ `total number o~ `exon counts`
##   &lt;chr&gt; &lt;chr&gt;     &lt;chr&gt;            &lt;dbl&gt; &lt;chr&gt;                    &lt;dbl&gt;
## 1 trea~ treated   sing~                5 35158667              15679615
## 2 trea~ treated   pair~                2 12242535 (x2)         15620018
## 3 trea~ treated   pair~                2 12443664 (x2)         12733865
## 4 untr~ untreated sing~                2 17812866              14924838
## 5 untr~ untreated sing~                6 34284521              20764558
## 6 untr~ untreated pair~                2 10542625 (x2)         10283129
## 7 untr~ untreated pair~                2 12214974 (x2)         11653031
## # ... with 2 more variables: `condition &lt;- factor(condition, levels =
## #   c(&quot;untreated&quot;, &quot;treated&quot;))` &lt;fct&gt;, `type &lt;- factor(sub(&quot;-.*&quot;, &quot;&quot;, type),
## #   levels = c(&quot;single&quot;, &quot;paired&quot;))` &lt;fct&gt;</code></pre>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1"></a><span class="kw">with</span>(pasillaSampleAnno,</span>
<span id="cb10-2"><a href="#cb10-2"></a>       <span class="kw">table</span>(condition, type))</span></code></pre></div>
<pre><code>##            type
## condition   paired-end single-read
##   treated            2           1
##   untreated          2           2</code></pre>
</div>
<div id="deseq2---das-r-paket-für-die-analyse-von-rna-seq-daten" class="slide section level2">
<h1>DESeq2 - DAS R Paket für die Analyse von RNA-seq Daten</h1>
<p>Das DESeq2 gehört zu den weltweit meist genutzen Pakten zur Analyse von RNA-seq Datensätzen.</p>
<p>Bevor wir mit der Analyse starten können, gilt es aus der Matrix <code>counts</code> und dem Dataframe der die Sampleinfo beschreibt <code>pasillaSampleAnno</code> ein sogenanntes <code>DESeqDataSet</code> zu erstellen:</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1"></a>mt &lt;-<span class="st"> </span><span class="kw">match</span>(<span class="kw">colnames</span>(counts), <span class="kw">sub</span>(<span class="st">&quot;fb$&quot;</span>, <span class="st">&quot;&quot;</span>, pasillaSampleAnno<span class="op">$</span>file))</span>
<span id="cb12-2"><a href="#cb12-2"></a><span class="kw">stopifnot</span>(<span class="op">!</span><span class="kw">any</span>(<span class="kw">is.na</span>(mt)))</span>
<span id="cb12-3"><a href="#cb12-3"></a></span>
<span id="cb12-4"><a href="#cb12-4"></a>pasilla &lt;-<span class="st"> </span><span class="kw">DESeqDataSetFromMatrix</span>(</span>
<span id="cb12-5"><a href="#cb12-5"></a>  <span class="dt">countData =</span> counts,</span>
<span id="cb12-6"><a href="#cb12-6"></a>  <span class="dt">colData   =</span> pasillaSampleAnno[mt, ],</span>
<span id="cb12-7"><a href="#cb12-7"></a>  <span class="dt">design    =</span> <span class="op">~</span><span class="st"> </span>condition)</span></code></pre></div>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1"></a>pasilla</span></code></pre></div>
<pre><code>## class: DESeqDataSet 
## dim: 14599 7 
## metadata(1): version
## assays(1): counts
## rownames(14599): FBgn0000003 FBgn0000008 ... FBgn0261574 FBgn0261575
## rowData names(0):
## colnames(7): untreated1 untreated2 ... treated2 treated3
## colData names(8): file condition ... condition &lt;- factor(condition,
##   levels = c(&quot;untreated&quot;, &quot;treated&quot;)) type &lt;- factor(sub(&quot;-.*&quot;, &quot;&quot;,
##   type), levels = c(&quot;single&quot;, &quot;paired&quot;))</code></pre>
</div>
<div id="durchführung-der-differentiellen-expressionsanalyse" class="slide section level2">
<h1>Durchführung der differentiellen Expressionsanalyse</h1>
<p>Ziel: Gene identifizieren, die zwischen treated und untreated Samples unterschiedlich sind. Wir wenden hier einen Hypothesentest an, der mit dem t-Test mathematisch verwandt ist.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1"></a>pasilla &lt;-<span class="st"> </span><span class="kw">DESeq</span>(pasilla)</span></code></pre></div>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a>pasilla</span></code></pre></div>
<pre><code>## class: DESeqDataSet 
## dim: 14599 7 
## metadata(1): version
## assays(4): counts mu H cooks
## rownames(14599): FBgn0000003 FBgn0000008 ... FBgn0261574 FBgn0261575
## rowData names(22): baseMean baseVar ... deviance maxCooks
## colnames(7): untreated1 untreated2 ... treated2 treated3
## colData names(9): file condition ... type &lt;- factor(sub(&quot;-.*&quot;, &quot;&quot;,
##   type), levels = c(&quot;single&quot;, &quot;paired&quot;)) sizeFactor</code></pre>
<p>Results</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1"></a>res &lt;-<span class="st"> </span><span class="kw">results</span>(pasilla)</span>
<span id="cb18-2"><a href="#cb18-2"></a>res[<span class="kw">order</span>(res<span class="op">$</span>padj), ] <span class="op">%&gt;%</span><span class="st"> </span>head</span></code></pre></div>
<pre><code>## log2 fold change (MLE): condition untreated vs treated 
## Wald test p-value: condition untreated vs treated 
## DataFrame with 6 rows and 6 columns
##                     baseMean    log2FoldChange              lfcSE
##                    &lt;numeric&gt;         &lt;numeric&gt;          &lt;numeric&gt;
## FBgn0039155 730.595806139728    4.619013535193  0.168706755496974
## FBgn0025111 1501.41051323996 -2.89986418283187  0.126920467532813
## FBgn0029167 3706.11653071978  2.19700028710382 0.0969888689039936
## FBgn0003360 4343.03539692487  3.17967242210135  0.143526226262858
## FBgn0035085 638.232608936723  2.56041205175815  0.137295199514033
## FBgn0039827 261.916235943103  4.16251610026134  0.232588756794067
##                          stat                pvalue                  padj
##                     &lt;numeric&gt;             &lt;numeric&gt;             &lt;numeric&gt;
## FBgn0039155  27.3789482915866 4.88534640286228e-165 4.06607381110228e-161
## FBgn0025111 -22.8478844996546 1.53390416993678e-115 6.38334220319191e-112
## FBgn0029167  22.6520869036895 1.33057630038134e-113 3.69146218269131e-110
## FBgn0003360  22.1539470861445 9.55682092175738e-109 1.98853551329467e-105
## FBgn0035085  18.6489553955341  1.28768628708899e-77  2.14348259348833e-74
## FBgn0039827   17.896463086334  1.25652932702639e-71  1.74301559814011e-68</code></pre>
<p>Im folgenden möchten wir die Results der differentiellen Expressionsanalyse mithilfe von Visualisierungen explorieren.</p>
</div>
<div id="histogramm-der-p-werte" class="slide section level2">
<h1>Histogramm der p-Werte</h1>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1"></a><span class="kw">ggplot</span>(<span class="kw">as</span>(res, <span class="st">&quot;data.frame&quot;</span>), <span class="kw">aes</span>(<span class="dt">x =</span> pvalue)) <span class="op">+</span></span>
<span id="cb20-2"><a href="#cb20-2"></a><span class="st">  </span><span class="kw">geom_histogram</span>(<span class="dt">binwidth =</span> <span class="fl">0.01</span>, <span class="dt">fill =</span> <span class="st">&quot;Royalblue&quot;</span>, <span class="dt">boundary =</span> <span class="dv">0</span>) <span class="op">+</span></span>
<span id="cb20-3"><a href="#cb20-3"></a><span class="st">    </span><span class="kw">theme_classic</span>()</span></code></pre></div>
<p><img src="pres_day5_files/figure-slidy/unnamed-chunk-11-1.png" width="432" /></p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1"></a><span class="kw">ggplot</span>(<span class="kw">as</span>(res, <span class="st">&quot;data.frame&quot;</span>), <span class="kw">aes</span>(<span class="dt">x =</span> pvalue)) <span class="op">+</span></span>
<span id="cb21-2"><a href="#cb21-2"></a><span class="st">  </span><span class="kw">geom_histogram</span>(<span class="dt">binwidth =</span> <span class="fl">0.005</span>, <span class="dt">fill =</span> <span class="st">&quot;Royalblue&quot;</span>, <span class="dt">boundary =</span> <span class="dv">0</span>) <span class="op">+</span></span>
<span id="cb21-3"><a href="#cb21-3"></a><span class="st">    </span><span class="kw">xlim</span>(<span class="kw">c</span>(<span class="dv">0</span>,<span class="fl">0.1</span>)) <span class="op">+</span></span>
<span id="cb21-4"><a href="#cb21-4"></a><span class="st">    </span><span class="kw">theme_classic</span>()</span></code></pre></div>
<p><img src="pres_day5_files/figure-slidy/unnamed-chunk-12-1.png" width="432" /></p>
</div>
<div id="ma-plot" class="slide section level2">
<h1>MA plot</h1>
<p>Fold change (M-value) vs. average counts (A-value)</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1"></a><span class="kw">plotMA</span>(pasilla, <span class="dt">ylim =</span> <span class="kw">c</span>( <span class="dv">-2</span>, <span class="dv">2</span>))</span></code></pre></div>
<p><img src="pres_day5_files/figure-slidy/unnamed-chunk-13-1.png" width="432" /></p>
<p>Roten Punkte = adjustierter P-Wert &lt; 0.1</p>
</div>
<div id="pca-plot" class="slide section level2">
<h1>PCA plot</h1>
<p>Die Principle Component Analysis (PCA) ist eine Methodik zur Dimensionsreduktion. Hierbei werden für jeden Sample die Infos zu allen 14,599 Genen auf zwei Dimensionen (x- und y-Achse) reduziert. Ein Sample ist ein Punkt auf dem Plot.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1"></a>pas_rlog &lt;-<span class="st"> </span><span class="kw">rlogTransformation</span>(pasilla)</span>
<span id="cb23-2"><a href="#cb23-2"></a><span class="kw">plotPCA</span>(pas_rlog, <span class="dt">intgroup=</span><span class="kw">c</span>(<span class="st">&quot;condition&quot;</span>, <span class="st">&quot;type&quot;</span>)) <span class="op">+</span><span class="st"> </span></span>
<span id="cb23-3"><a href="#cb23-3"></a><span class="st">    </span><span class="kw">coord_fixed</span>() <span class="op">+</span></span>
<span id="cb23-4"><a href="#cb23-4"></a><span class="st">    </span><span class="kw">theme_classic</span>()</span></code></pre></div>
<p><img src="pres_day5_files/figure-slidy/unnamed-chunk-14-1.png" width="432" /></p>
</div>
<div id="heatmap" class="slide section level2">
<h1>Heatmap</h1>
<p>Eine Heatmap aller Gene darzustellen macht keinen Sinn. Es gilt Filterkriterien zu definieren.</p>
<p>Filter 1) Top 30 most expressed genes.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1"></a>select &lt;-<span class="st"> </span><span class="kw">order</span>(<span class="kw">rowMeans</span>(<span class="kw">assay</span>(pas_rlog)), <span class="dt">decreasing =</span> <span class="ot">TRUE</span>)[<span class="dv">1</span><span class="op">:</span><span class="dv">30</span>]</span>
<span id="cb24-2"><a href="#cb24-2"></a></span>
<span id="cb24-3"><a href="#cb24-3"></a><span class="kw">pheatmap</span>( <span class="kw">assay</span>(pas_rlog)[select, ],</span>
<span id="cb24-4"><a href="#cb24-4"></a>     <span class="dt">scale =</span> <span class="st">&quot;row&quot;</span>,</span>
<span id="cb24-5"><a href="#cb24-5"></a>     <span class="dt">annotation_col =</span> <span class="kw">as.data.frame</span>(</span>
<span id="cb24-6"><a href="#cb24-6"></a>        <span class="kw">colData</span>(pas_rlog)[, <span class="kw">c</span>(<span class="st">&quot;condition&quot;</span>, <span class="st">&quot;type&quot;</span>)] ))</span></code></pre></div>
<p><img src="pres_day5_files/figure-slidy/unnamed-chunk-15-1.png" width="576" /></p>
<p>Filter 2) Adjusted p-Value &lt; 0.01 und absolute log2 Fold Change &gt; 2</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1"></a>select2 &lt;-<span class="st"> </span>(res<span class="op">$</span>padj<span class="op">&lt;</span><span class="fl">0.01</span> <span class="op">&amp;</span><span class="st"> </span><span class="kw">abs</span>(res<span class="op">$</span>log2FoldChange)<span class="op">&gt;</span><span class="dv">2</span>)<span class="op">*</span><span class="dv">1</span></span>
<span id="cb25-2"><a href="#cb25-2"></a></span>
<span id="cb25-3"><a href="#cb25-3"></a><span class="kw">pheatmap</span>( <span class="kw">assay</span>(pas_rlog)[select2 <span class="op">%in%</span><span class="st"> </span><span class="dv">1</span>, ],</span>
<span id="cb25-4"><a href="#cb25-4"></a>     <span class="dt">scale =</span> <span class="st">&quot;row&quot;</span>,</span>
<span id="cb25-5"><a href="#cb25-5"></a>     <span class="dt">annotation_col =</span> <span class="kw">as.data.frame</span>(</span>
<span id="cb25-6"><a href="#cb25-6"></a>        <span class="kw">colData</span>(pas_rlog)[, <span class="kw">c</span>(<span class="st">&quot;condition&quot;</span>, <span class="st">&quot;type&quot;</span>)] ))</span></code></pre></div>
<p><img src="pres_day5_files/figure-slidy/unnamed-chunk-16-1.png" width="576" /></p>
</div>
<div id="export-der-results-als-csv-file" class="slide section level2">
<h1>Export der Results als csv File</h1>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1"></a><span class="kw">write.csv</span>(<span class="kw">as.data.frame</span>(res), <span class="dt">file =</span> <span class="st">&quot;treated_vs_untreated.csv&quot;</span>)</span></code></pre></div>
</div>
<div id="übung" class="slide section level2">
<h1>Übung</h1>
<p>Die Übung des heutigen Tages besteht im Ausführen und Nachvollziehen des R Codes der Slides.</p>
</div>

  <!-- dynamically load mathjax for compatibility with self-contained -->
  <script>
    (function () {
      var script = document.createElement("script");
      script.type = "text/javascript";
      script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
      document.getElementsByTagName("head")[0].appendChild(script);
    })();
  </script>

</body>
</html>
