<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}

.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">AM</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="refs.html">Publications</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Software
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="CVE.html">CVE</a>
    </li>
    <li>
      <a href="MetaboDiff.html">MetaboDiff</a>
    </li>
  </ul>
</li>
<li>
  <a href="teaching.html">R-Kurs</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/andreasmock">
    <span class="fa fa-github-square"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/am0ck">
    <span class="fa fa-twitter"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<p><br></p>
<div id="section" class="section level3">
<h3>2018</h3>
<p><a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty344/4987147">MetaboDiff: an R package for differential metabolomic analysis</a><br />
<strong>Mock A</strong>, Warta R, Dettling S, Brors B, Jäger D, Herold-Mende C.<br />
<strong>Bioinformatics.</strong> 2018 Apr 26. doi: 10.1093/bioinformatics/bty344. [Epub ahead of print.]</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/29227286">Drug-perturbation-based stratification of blood cancer.</a><br />
Dietrich S, Oleś M, Lu J, Sellner L, Anders S, Velten B, Wu B, Hüllein J, da Silva Liberio M, Walther T, Wagner L, Rabe S, Ghidelli-Disse S, Bantscheff M, Oleś AK, Słabicki M, <strong>Mock A</strong>, Oakes CC, Wang S, Oppermann S, Lukas M, Kim V, Sill M, Benner A, Jauch A, Sutton LA, Young E, Rosenquist R, Liu X, Jethwa A, Lee KS, Lewis J, Putzker K, Lutz C, Rossi D, Mokhir A, Oellerich T, Zirlik K, Herling M, Nguyen-Khac F, Plass C, Andersson E, Mustjoki S, von Kalle C, Ho AD, Hensel M, Dürig J, Ringshausen I, Zapatka M, Huber W, Zenz T.<br />
<strong>J Clin Invest.</strong> 2018 Jan 2;128(1):427-445. doi: 10.1172/JCI93801. Epub 2017 Dec 11.</p>
<p><br></p>
</div>
<div id="section-1" class="section level3">
<h3>2017</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/28901970">The 2016 WHO classification of central nervous system tumors: what neurologists need to know</a><br />
DeWitt JC*, <strong>Mock A*</strong>, Louis DN.<br />
<strong>Curr Opin Neurol</strong>. 2017 Dec;30(6):643-649. doi: 10.1097/WCO.0000000000000490.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/28545463">CVE: an R package for interactive variant prioritisation in precision oncology</a><br />
<strong>Mock A</strong>, Murphy S, Morris J, Marass F, Rosenfeld N, Massie C.<br />
<strong>BMC Med Genomics</strong>. 2017 May 25;10(1):37. doi: 10.1186/s12920-017-0261-6.</p>
<p><br></p>
</div>
<div id="section-2" class="section level3">
<h3>2016</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/27782828">Spatial transcriptome analysis reveals Notch pathway-associated prognostic markers in IDH1 wild-type glioblastoma involving the subventricular zone</a><br />
Jungk C, <strong>Mock A</strong>, Exner J, Geisenberger C, Warta R, Capper D, Abdollahi A, Friauf S, Lahrmann B, Grabe N, Beckhove P, von Deimling A, Unterberg A, Herold-Mende C.<br />
<strong>BMC Med</strong>. 2016 Oct 26;14(1):170.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/27344556">Prognostic value of the extent of resection in supratentorial WHO grade II astrocytomas stratified for IDH1 mutation status: a single-center volumetric analysis.</a><br />
Jungk C, Scherer M, <strong>Mock A</strong>, Capper D, Radbruch A, von Deimling A, Bendszus M, Herold-Mende C, Unterberg A.<br />
<strong>J Neurooncol</strong>. 2016 Sep;129(2):319-28. doi: 10.1007/s11060-016-2177-y.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/27069124">Combined Treatment of ATRA with Epigenetic Drugs Increases Aggressiveness of Glioma Xenografts.</a><br />
Schmoch T, Gal Z, <strong>Mock A</strong>, Mossemann J, Lahrmann B, Grabe N, Schmezer P, Lasitschka F, Beckhove P, Unterberg A, Herold-Mende C.<br />
<strong>Anticancer Res</strong>. 2016 Apr;36(4):1489-96.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/26934681">LOC283731 promoter hypermethylation prognosticates survival after radiochemotherapy in IDH1 wild-type glioblastoma patients.</a><br />
<strong>Mock A*</strong>, Geisenberger C*, Orlik C, Warta R, Schwager C, Jungk C, Dutruel C, Geiselhart L, Weichenhan D, Zucknick M, Nied AK, Friauf S, Exner J, Capper D, Hartmann C, Lahrmann B, Grabe N, Debus J, von Deimling A, Popanda O, Plass C, Unterberg A, Abdollahi A, Schmezer P, Herold-Mende C.<br />
<strong>Int J Cancer</strong>. 2016 Jul 15;139(2):424-32. doi: 10.1002/ijc.30069. Epub 2016 Mar 18.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/26894859">Transcriptomic analysis of aggressive meningiomas identifies PTTG1 and LEPR as prognostic biomarkers independent of WHO grade.</a><br />
Schmidt M*, <strong>Mock A*</strong>, Jungk C, Sahm F, Ull AT, Warta R, Lamszus K, Gousias K, Ketter R, Roesch S, Rapp C, Schefzyk S, Urbschat S, Lahrmann B, Kessler AF, Löhr M, Senft C, Grabe N, Reuss D, Beckhove P, Westphal M, von Deimling A, Unterberg A, Simon M, Herold-Mende C.<br />
<strong>Oncotarget</strong>. 2016 Mar 22;7(12):14551-68. doi: 10.18632/oncotarget.7396.</p>
<p><br></p>
</div>
<div id="section-3" class="section level3">
<h3>2015</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25931051">Molecular profiling of long-term survivors identifies a subgroup of glioblastoma characterized by chromosome 19/20 co-gain.</a><br />
Geisenberger C*, <strong>Mock A*</strong>, Warta R, Rapp C, Schwager C, Korshunov A, Nied AK, Capper D, Brors B, Jungk C, Jones D, Collins VP, Ichimura K, Bäcklund LM, Schnabel E, Mittelbron M, Lahrmann B, Zheng S, Verhaak RG, Grabe N, Pfister SM, Hartmann C, von Deimling A, Debus J, Unterberg A, Abdollahi A, Herold-Mende C.<br />
<strong>Acta Neuropathol</strong>. 2015 Sep;130(3):419-34. doi: 10.1007/s00401-015-1427-y. Epub 2015 May 1.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/27057435">Non-invasive glioblastoma immunoprofiling by printed peptide arrays.</a><br />
<strong>Mock A</strong>, Herold-Mende C.<br />
<strong>Oncoimmunology</strong>. 2015 Aug 12;5(2):e1069941. eCollection 2016 Feb.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25964546">Endothelial Cells Derived from Non-malignant Tissues Are of Limited Value as Models for Brain Tumor Vasculature.</a><br />
Lohr J*, <strong>Mock A*</strong>, Beckhove P, Herold-Mende C.<br />
<strong>Anticancer Res</strong>. 2015 May;35(5):2681-90.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25944688">Printed peptide arrays identify prognostic TNC serumantibodies in glioblastoma patients.</a><br />
<strong>Mock A</strong>, Warta R, Geisenberger C, Bischoff R, Schulte A, Lamszus K, Stadler V, Felgenhauer T, Schichor C, Schwartz C, Matschke J, Jungk C, Ahmadi R, Sahm F, Capper D, Glass R, Tonn JC, Westphal M, von Deimling A, Unterberg A, Bermejo JL, Herold-Mende C.<br />
<strong>Oncotarget</strong>. 2015 May 30;6(15):13579-90.</p>
<p><br></p>
</div>
<div id="section-4" class="section level3">
<h3>2014</h3>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25175691">Microenvironment and brain tumor stem cell maintenance: impact of the niche.</a><br />
Herold-Mende C, <strong>Mock A</strong>.<br />
<strong>Anticancer Agents Med Chem</strong>. 2014;14(8):1065-74. Review.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/24740185">Reduced promoter methylation and increased expression of CSPG4 negatively influences survival of HNSCC patients.</a><br />
Warta R, Herold-Mende C, Chaisaingmongkol J, Popanda O, <strong>Mock A</strong>, Mogler C, Osswald F, Herpel E, Küstner S, Eckstein V, Plass C, Plinkert P, Schmezer P, Dyckhoff G.<br />
<strong>Int J Cancer</strong>. 2014 Dec 1;135(11):2727-34. doi: 10.1002/ijc.28906. Epub 2014 Apr 26.</p>
<p><br></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/23530497">Lessons we learned from high-throughput and top-down systems biology analyses about glioma stem cells.</a><br />
<strong>Mock A</strong>, Chiblak S, Herold-Mende C.<br />
<strong>Curr Pharm Des</strong>. 2014;20(1):66-72. Review.</p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
